template-helpers/hbs-helpers.js

```diff
@@ -21,6 +21,25 @@ function getSupportedLocales(args) {
 }
 
 
+function englishInAcceptLanguages(args) {
+  const acceptedLanguages = args.data.root.req.acceptsLanguages();
+  return acceptedLanguages.some(locale => locale.startsWith("en"));
+}
+
+
+function recruitmentBanner(args) {
+  if (!AppConstants.RECRUITMENT_BANNER_LINK || !AppConstants.RECRUITMENT_BANNER_TEXT) {
+    return;
+  }
+
+  if (!englishInAcceptLanguages(args)) {
+    return;
+  }
+
+  return `<div class="recruitment-banner"><a id="recruitment-banner" href="${AppConstants.RECRUITMENT_BANNER_LINK}"  target="_blank" rel="noopener noreferrer" data-ga-link="" data-event-category="Recruitment" data-event-label="${AppConstants.RECRUITMENT_BANNER_TEXT}">${AppConstants.RECRUITMENT_BANNER_TEXT}</a></div>`;
```

Comment:
Is it OK that `data-event-label` will have spaces and other stuff?

Also, are we expecting that `RECRUITMENT_BANNER_TEXT` is always just a simple string and will never include markup (like a `<br>` or `<i>` or `<strong>`), or any double-quotes (which would probably just throw up right around this attribute -- since we aren't doing any escaping).

Reply:
I'll have it escape `"` and `'` chars here and we'll have to remember this about the text.